### PR TITLE
Delete api for inventory

### PR DESCRIPTION
**Pull request by django9847**

### CODE REVIEW

File: service/routes.py [hunk 1]
```diff
@@ -26,12 +26,27 @@ def index():
...
+def delete_inventory(product_id, condition):
+    '''This endpoint will delete a product with the specified id and condition'''
+    app.logger.info("Request to delete a product with product_id %s and condition %s", product_id, condition)
+    product = Inventory.query.filter_by(product_id=product_id, condition=condition).first()
+    if product:
+        product.delete()
+        # db.session.commit()
+        app.logger.info("Product with product_id %s and condition %s deleted.", product_id, condition)
+        return ("", status.HTTP_204_NO_CONTENT)
+    return ("", status.HTTP_404_NOT_FOUND)
```
Comment (MarvinTellis) [inline service/routes.py:48]: The default response code should be `204 no content`, even if the item to be deleted was not present in the DB

Comment (django9847) [reply service/routes.py:48]: @MarvinTellis Should I go for 204 or 404. I am bit confused here as per the API contract which mentions the following. 

"Given a request to delete a non-existent item from the inventory,
When the API is called with the appropriate parameters,
Then the API should respond with a status code of 404 (Not Found).
And the response body should contain an error message indicating that the specified shopcart item was not found.
The format should look like this:

{
    <error> : 404
    <error_code> : Not Found
} "

Comment (django9847) [reply service/routes.py:48]: Okay, so we are going for 204 ? Should I make the change ?

Comment (MarvinTellis) [reply service/routes.py:48]: Yes. I have already updated the story on Kanban board. You can make the changes in code

Comment (django9847) [reply service/routes.py]: Done. Thank you

**Review by MarvinTellis — APPROVED** [review]

File: service/routes.py [hunk 2]
```diff
@@ -26,12 +26,25 @@ def index():
...
 ######################################################################
 
+######################################################################
+# DELETE A INVENTORY ITEM
+######################################################################
+@app.route("/inventory/<int:product_id>/<string:condition>", methods=["DELETE"])
+def delete_inventory(product_id, condition):
+    '''This endpoint will delete a product with the specified id and condition'''
+    app.logger.info("Request to delete a product with product_id %s and condition %s", product_id, condition)
+    product = Inventory.query.filter_by(product_id=product_id, condition=condition).first()
```
Comment (IrregularHunterX) [inline service/routes.py:42]: Just wondering as to why Inventory.find(...) isn't going to work here?

**Review by IrregularHunterX — APPROVED** [review]

**Review by vkrishna1997 — CHANGES_REQUESTED** [review]

File: service/routes.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 List, Create, Read, Update, and Delete products from the inventory database
 """
-
+from service.models import Inventory, Condition
```
[the remark this answers is not in the document]
Comment (django9847) [reply service/routes.py:6]: removed

File: service/routes.py (hunk 2 above)
Comment (django9847) [reply service/routes.py]: Yes, both are same, internally find does the same. Changed to find() for code uniformity.

**Review by IrregularHunterX — APPROVED** [review]

**Pull request merged**